src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -244,6 +244,13 @@ def run_test(
         :param tick_duration_s: length of a tick
         :param index: index of the current test. default is 0 (invariant test)
                       values can be passed in during aggregate testing for different timeout durations
+        :param ci_cmd_with_delay: A list consisting of a duration, and a
+                        tuple forming a ci command
```

Comment:
```suggestion
        :param ci_cmd_with_delay: A list consisting of tuples with a duration and ci command
                        
```
I meant changing comment docs like this.
Also do to line 110